dev-haskell/hashable/hashable-1.3.1.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+#hackport: flags: sse41:cpu_flags_x86_sse4_1,sse2:cpu_flags_x86_sse2,integer-gmp:gmp
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="A class for types that can be converted to a hash value"
+HOMEPAGE="https://github.com/haskell-unordered-containers/hashable"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="+gmp"
+
+RDEPEND=">=dev-haskell/text-0.12:=[profile?] <dev-haskell/text-1.3:=[profile?]
+	>=dev-lang/ghc-8.4.3:=
+	!gmp? ( dev-haskell/integer-simple:=[profile?] )
```

Comment:
`dev-haskell/integer-simple` should never be present in dependencies explicitly. It should automatically be picked by ghc's build flavour or just force gmp.